segmentation_processing/start_server.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+#SBATCH --account=adam
+#SBATCH --partition=adam
+#SBATCH --ntasks=1
+#SBATCH --mem-per-cpu=8G
+#SBATCH --cpus-per-task=2
+#SBATCH --gpus-per-task=1
+#SBATCH --job-name=STEGO_API
+#SBATCH --output=./logs/%x-%j.out    # %x-%j means JOB_NAME-JOB_ID.
+
+set -euo pipefail
+
+# Indefinite timeout used to keep the stego_model in memory for a long time
+
+umask 0002  # Allow files to be created as group-writable
+../venv/bin/gunicorn \
+  --config ./gunicorn.conf.py \
+  --access-logfile ./server.log \
```

Comment:
Right, there's also a 'small' problem with this script which is we don't use a `venv` rather that invocation to gunicorn needs to be updated to point at someone's environment on the cluster... 

Ideally we'd create a shared environment on the the `nas` for this script to use but that may be easier to manually configure than create a script for.

Reply:
Ah, I totally glossed over the `venv`. As far as manually configuring -- you mean activate this before running the script?

Would it be straightforward to run this in the same env as other preprocessing code? I guess that's a question for me to answer as well 🙂 -- if I just install Gunicorn/necessary other things, does it work.

Reply:
The requirements file in this directory has what I think needs to be installed for the server to run. It may work in another environment I just only tested in a new one for simplicity. 

On the cluster my submit script points to the `gunicorn` bin file in my conda environment. We may be able to run this by just activating the environment `conda activate` and running `gunicorn` on the command line, I didn't test that. I just modified the scripts LESTAT uses.